### PR TITLE
Add network_token destination. Fix challenge indicator and exemption

### PR DESCRIPTION
**Pull request by armando-rodriguez-cko**
## Changes
- Add network_token destination request
- Fix challenge indicator and exemption

## API reference
https://api-reference.checkout.com/#operation/requestAPaymentOrPayout!path=2/destination&t=request

### CODE REVIEW

**Review by portothree — APPROVED** [review]

File: src/main/java/com/checkout/common/ChallengeIndicator.java [hunk 1]
```diff
@@ -8,19 +8,9 @@ public enum ChallengeIndicator {
     CHALLENGE_REQUESTED,
     @SerializedName("challenge_requested_mandate")
     CHALLENGE_REQUESTED_MANDATE,
-    @SerializedName("data_share")
```
Comment (ogaragatyi) [inline src/main/java/com/checkout/common/ChallengeIndicator.java:11]: Hey. Alex from Wise lurking here 👋 
Wondering if this is considered a breaking change?

Comment (armando-rodriguez-cko) [reply src/main/java/com/checkout/common/ChallengeIndicator.java:11]: I checked it, I made this mistake adding this string here. But I can leave it

Comment (armando-rodriguez-cko) [reply src/main/java/com/checkout/common/ChallengeIndicator.java:11]: I leave the correct values https://api-reference.checkout.com/#operation/requestAPaymentOrPayout!path=0/3ds/2/challenge_indicator&t=request

Comment (armando-rodriguez-cko) [reply src/main/java/com/checkout/common/ChallengeIndicator.java:11]: Please @ogaragatyi resolve this conversation

Comment (ogaragatyi) [reply src/main/java/com/checkout/common/ChallengeIndicator.java:11]: Weird think - I resolved my comment but not the whole thread. See this kind of behavior from Github for the first time

**Pull request merged**